src/plugins/graphql-operation-logging-plugin.ts

```diff
@@ -80,7 +85,10 @@ export function graphqlOperationLoggingPlugin<TContext extends GraphQLContext<TL
         ctx: GraphQLRequestContextWillSendResponse<TContext>,
         subsequentPayload?: GraphQLExperimentalFormattedSubsequentIncrementalExecutionResult,
       ) {
-        const { started, logger } = contextValue
+        const { started, logger } = {
+          ...contextValue,
+          ...(resolveCustomLogger ? { logger: resolveCustomLogger(contextValue) } : {}),
+        }
```

Comment:
Very much a nit, I think separate declarations could make it easier to read.

```suggestion
        const { started } = contextValue
        const { logger } = resolveCustomLogger
          ? { logger: resolveCustomLogger(contextValue) }
          : contextValue
```

Reply:
Yeah, no problems. I had umh'd about that a bit myself